### PR TITLE
Add bxt_tas_editor_set_frametime

### PR DESCRIPTION
**Pull request by khanghugo**

### CODE REVIEW

File: BunnymodXT/hud_custom.cpp [hunk 1]
```diff
@@ -1342,6 +1344,14 @@ namespace CustomHud
 
 			out << "Health: " << frame_bulk_status.health << '\n';
 			out << "Armor: " << frame_bulk_status.armor << '\n';
+
+			if (!(!frame_bulk_status.frametime.compare("0.010000001") ||
+					!frame_bulk_status.frametime.compare("0.001") || 
+					!frame_bulk_status.frametime.compare("0.002") || 
+					!frame_bulk_status.frametime.compare("0.004")
+					)) {
```
Comment (YaLTeR) [inline BunnymodXT/hud_custom.cpp:1352]: Why the condition? I think just print it maybe

File: BunnymodXT/modules/HwDLL.cpp [hunk 2]
```diff
@@ -3578,6 +3580,23 @@ struct HwDLL::Cmd_BXT_TAS_Editor_Toggle
...
 
+struct HwDLL::Cmd_BXT_TAS_Editor_Set_Frametime
+{
+	USAGE("Usage: bxt_tas_editor_set_frametime <frametime>\n Sets frametime on the currently selected point.\n");
+
+	static void handler(const char *value)
+	{
+		auto& hw = HwDLL::GetInstance();
+
+		if (boost::lexical_cast<double>(value) <= 0.f) {
```
Comment (YaLTeR) [inline BunnymodXT/modules/HwDLL.cpp:3591]: Can we avoid including `lexical_cast` just for this? Maybe `atof`, or honestly just `!isdigit(value[0])`

File: BunnymodXT/hud_custom.cpp (hunk 1 above)
Comment (khanghugo) [reply BunnymodXT/hud_custom.cpp:1352]: Hmm I thought to not add anything more to the TAS editor status as there is too much. Sure, I will just have it there.

File: BunnymodXT/modules/HwDLL.cpp (hunk 2 above)
Comment (khanghugo) [reply BunnymodXT/modules/HwDLL.cpp:3591]: Thanks. I don't know how float works and concerned that 0.01000001 thing might get lost. Not anymore.

**Comment by YaLTeR** [comment]
Thanks

**Pull request merged**